app/assets/stylesheets/home.scss

```diff
@@ -159,14 +174,26 @@ html, body {
   text-align: left;
 }
 
+.price p{
+  margin: 10px 0;
+}
+
 .pricing-table .bullet-item {
   text-align: left;
   padding-left: 20px;
   padding-right: 20px;
 }
 
-h2 {
-  margin-top: 0;
+#featured h1 {
+  margin: 65px 0 0 30px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+#featured #index-header {
+  margin: 15px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `15px 0 0` instead of `15px 0 0 0`